src/components/Calculator.js

```diff
@@ -0,0 +1,82 @@
+import '../css/calculator.css';
+import React from 'react';
+
+function Calculator() {
+  return (
+    <div>
+      <div className="main-container">
+        <div className="screen">0</div>
+
+        <input className="all-btn" id="AC-btn" type="button" value="AC" />
+
+        <input
+          className="all-btn"
+          id="plus-or-minus-btn"
+          type="button"
+          value="+/-"
+        />
+
+        <input className="all-btn" id="percent-btn" type="button" value="%" />
+
+        <input
+          className="orange-btn all-btn"
+          id="division-btn"
+          type="button"
+          value="÷"
+        />
+
+        <input className="all-btn" id="seven-btn" type="button" value="7" />
+
+        <input className="all-btn" id="eight-btn" type="button" value="8" />
+
+        <input className="all-btn" id="nine-btn" type="button" value="9" />
```

Comment:
- [ ] your calculator looks nice but it would be better if `Calculator.js` components have at least one child component in order so that it will be reuseable through the the app 
- Screenshot of the requirements 
<img width="315" alt="Screenshot 2023-05-02 at 23 19 12" src="https://user-images.githubusercontent.com/61790587/235798229-fba3b646-fc63-43a6-a163-4ec6e9d2df4c.png">